applications/skypuff/skypuff_conf.h

```diff
@@ -0,0 +1,223 @@
+#ifndef CONF_SKYPUFF_H_
+#define CONF_SKYPUFF_H_
+
+#include "datatypes.h"
+
+typedef struct
```

Comment:
Need a comment, that's part of mc_conf which will be sent to Skypuff controller app.